Octokit/Http/ApiInfo.cs

```diff
@@ -51,5 +51,25 @@ public class ApiInfo
         /// Information about the API rate limit
         /// </summary>
         public RateLimit RateLimit { get; private set; }
+
+        /// <summary>
+        /// Allows you to clone ApiInfo 
+        /// </summary>
+        /// <returns>A clone of <seealso cref="ApiInfo"/></returns>
+        public ApiInfo Clone()
+        {
+            // Seem to have to do this to pass a whole bunch of tests (for example Octokit.Tests.Clients.EventsClientTests.DeserializesCommitCommentEventCorrectly)
+            // I believe this has something to do with the Mocking framework.
+            if (this == null || this.Links == null || this.OauthScopes == null || this.RateLimit == null || this.Etag == null)
```

Comment:
Fair point.

I really not sure why I need this line at all - but something blew up within the mocking framework so I added everything and kitchen sink.

Maybe went a tad overboard ;)